tests/test_rio_pyramid.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import sys
+
+from click.testing import CliRunner
+
+import rasterio
+from rasterio.rio import pyramid
+
+
+logging.basicConfig(stream=sys.stderr, level=logging.DEBUG)
+
+
+def test_build(data):
+    runner = CliRunner()
+    inputfile = str(data.join('RGB.byte.tif'))
+    result = runner.invoke(pyramid.pyramid, [inputfile, '--build', '2,4,8'])
+    assert result.exit_code == 0
+    with rasterio.open(inputfile) as src:
+        assert src.overviews(1) == [2, 4, 8]
+        assert src.overviews(2) == [2, 4, 8]
+        assert src.overviews(3) == [2, 4, 8]
+
+
+def test_build(data):
```

Comment:
@sgillies this needs a unique name; pytest is testing this one and ignoring the prior because they match names.

You also need a test of invalid input format to get coverage of the Exception case in ```build_handler```...